var/spack/repos/builtin/packages/samrai/package.py

```diff
@@ -66,11 +66,10 @@ class Samrai(AutotoolsPackage):
     depends_on('hdf5+mpi')
     depends_on('m4', type='build')
 
-    # At some point later versions of boost were not able to be found
-    # by SAMRAI during configure, so we're using boost <= 1.60.0 for
-    # < 3.12.0 versions. I don't know what version of boost that
-    # happened at though without trying each version of boost.
-    depends_on('boost@:1.60.0', when='@0:3.11.99')
+    # From 3.12.0, samrai no long depends on boost.
```

Comment:
Typo: "long" -> "longer"

I would also switch "From" to "Starting with"

Reply:
Sure. 
Should I resubmit a new PR?

Reply:
Nah, just add a second commit to this PR.